### PR TITLE
Make competible with alchemlyb 2

### PR DESCRIPTION
**Pull request by xiki-tempula**

### CODE REVIEW

File: python/BioSimSpace/Sandpit/Exscientia/FreeEnergy/_relative.py [hunk 1]
```diff
@@ -36,6 +35,8 @@
 import zipfile as _zipfile
 from glob import glob as _glob
 
+import math as _math
```
Comment (msuruzhon) [inline python/BioSimSpace/Sandpit/Exscientia/FreeEnergy/_relative.py]: Why have you changed the order?

File: python/BioSimSpace/Sandpit/Exscientia/FreeEnergy/_relative.py [hunk 2]
```diff
@@ -642,7 +643,7 @@ def _analyse_noSOMD(
             units="kcal/mol",
             software=engine,
             dir=work_dir,
-            prefix=prefix,
```
Comment (msuruzhon) [inline python/BioSimSpace/Sandpit/Exscientia/FreeEnergy/_relative.py:645]: So is that the main change in behaviour between 1 and 2?

Comment (xiki-tempula) [reply python/BioSimSpace/Sandpit/Exscientia/FreeEnergy/_relative.py:645]: Yes, the `dir` accepts a real directory so `'lambda_*'` is not accepted as `dir` input and has to go to `prefex`

File: python/BioSimSpace/Sandpit/Exscientia/FreeEnergy/_relative.py (hunk 1 above)
Comment (xiki-tempula) [reply python/BioSimSpace/Sandpit/Exscientia/FreeEnergy/_relative.py]: pycharm did it for me, let me recreate the env and run it again.

Comment (xiki-tempula) [reply python/BioSimSpace/Sandpit/Exscientia/FreeEnergy/_relative.py]: It is back now

**Review by msuruzhon — APPROVED** [review]
LGTM

**Pull request merged**